### PR TITLE
ECIL-176 - Use importcontrols email address within sanctions

### PR DESCRIPTION
**Pull request by chris-pettinga**
Creating a new template for Sanctions RFI requests that uses the sanctions email address rather than the generic ILB contact address.

### CODE REVIEW

File: web/migrations/0047_auto_20241113_1725.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        template_name="Further Information Request Email",
+        template_type=Template.EMAIL_TEMPLATE,
+        application_domain=Template.IMPORT_APPLICATION,
+        template_code=TemplateCodes.IMA_SANCTIONS_RFI,
+    )
+
+    TemplateVersion.objects.create(
+        template=new_template,
+        title="Request for more information [[CASE_REFERENCE]]",
+        content=f"Dear [[IMPORTER_NAME]], You need to provide some more information for Import Licencing Branch (ILB) to process your application. [DESCRIBE WHAT INFORMATION OR CLARIFICATION IS NEEDED. INCLUDE SUGGESTIONS IF RELEVANT] regarding [DESCRIBE WHAT FURTHER INFORMATION IS NEEDED / WHAT IS UNCLEAR, MAKE SUGGESTIONS IF RELEVANT]. Your application will not be processed further until you respond to this request. The application will be closed if the requested response is not received within 1 working day of this email. Contact {settings.ICMS_SANCTIONS_EMAIL} if you have any questions about your application. Include case reference number [[CASE_REFERENCE]] in your email, so we know which application you are contacting us about. Yours sincerely, [[CASE_OFFICER_NAME]]",
```
Comment (marcuspp) [inline web/migrations/0047_auto_20241113_1725.py:21]: Does this need `\n\n` to format the copy?

File: web/migrations/0047_auto_20241113_1725.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.2.16 on 2024-11-13 17:25
```
Comment (MattHolmes123) [inline web/migrations/0047_auto_20241113_1725.py:1]: Can you give this a better name please.

File: web/migrations/0047_auto_20241113_1725.py (hunk 1 above)
Comment (chris-pettinga) [reply web/migrations/0047_auto_20241113_1725.py]: yes done

File: web/migrations/0047_auto_20241113_1725.py (hunk 2 above)
Comment (chris-pettinga) [reply web/migrations/0047_auto_20241113_1725.py]: yes done

File: web/migrations/0047_create_sanctions_rfi_template.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# Generated by Django 4.2.16 on 2024-11-13 17:25
```
Comment (marcuspp) [inline web/migrations/0047_create_sanctions_rfi_template.py:1]: The file should be `0046` not `0047`

File: web/management/commands/fixtures/templates.json [hunk 4]
```diff
@@ -1178,6 +1178,19 @@
...
+    {
+        "model": "web.template",
+        "pk": 93,
+        "fields": {
+            "is_active": true,
+            "template_name": "Further Information Request Email",
+            "template_code": "IMA_SANCTIONS_RFI",
+            "template_type": "EMAIL_TEMPLATE",
+            "application_domain": "IMA",
+            "country_translation_set": null,
```
Comment (marcuspp) [inline web/management/commands/fixtures/templates.json:1190]: Consider adding sanctions to the template name 

Screenshot 2024-11-22 at 10 02 51

Viewing them in the admin area. Makes it hard to see which template is which

**Review by marcuspp — APPROVED** [review]

**Review by mikeylockwood — APPROVED** [review]

**Pull request merged**